### PR TITLE
These are the original files for olivia starnes with the updates hopefully.

### PR DESCRIPTION
**Pull request by Starnol**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
```
Comment (bjones1) [inline index.rst:1]: Delete this file.

File: Thor Method.py [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+# This game has a built in code for the directions that you can't see. After trail and error, I figured out that all it is doing is either increasing/decreasing x and/or y. It is **not** updating Thor's position. Which can cause him to go outside of the boundry. 
+#
+
+from re import X
+import sys
+import math
+
+# *Auto- generated code below aims to helping you parse the standard input according to the problem statement. Hint: you can use the debug stream to print initialTX and initialTY, if Thor seems not to follow your instructions.*
+ # **Light_x**: the X position of the light of power
+ #
```
Comment (bjones1) [inline Thor Method.py:43]: Omit the blank lines between item and use either an unordered list or definition list:

```
# :light_x: the X position of the light of power
# :light_y: the Y position of the light of power
```

File: Thor Method.py [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+ # **Light_y**: the Y position of the light of power 
+ #
+ # **Initial_tx**: Thor's starting X position
+ #
+ # **Initial_ty**: Thor's starting Y position
+ #
+light_x, light_y, initial_tx, initial_ty = [int(i) for i in input().split()]
```
Comment (bjones1) [inline Thor Method.py:50]: Explain -- why `int`? What does `split` do? Why `[stuff]` instead of `stuff`?

File: Thor Method.py [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+
+# game loop
```
Comment (bjones1) [inline Thor Method.py:52]: Make this a heading

File: Thor Method.py [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+while True:
+    remaining_turns = int(input())  # The remaining amount of turns Thor can move. Do not remove this line.
+
+    # Write an action using print
+    # To debug: print("Debug messages...", file=sys.stderr, flush=True)
+
+
+    # A single line providing the move to be made: N NE E SE S SW W or NW
+    print("SE")
```
Comment (bjones1) [inline Thor Method.py:61]: Why this this? You always move SE?

File: Thor Method.py [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+
+#   
+# Solution: First Try with C++
```
Comment (bjones1) [inline Thor Method.py:64]: Just talk about your final code, not the intermediate steps -- at this point, everything is Python, correct?

File: Thor Method.py [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+# My solution code is in C++. Please click `here <https://www.softwaretestinghelp.com/loops-in-cpp/#:~:text=Loops%20In%20C%2B%2B%201%20For%20Loop.%20The%20construct,...%205%20Infinite%20Loop.%20...%206%20Conclusion.%20>`_ if you are not familiar with it. 
+#
+# My Thoughts and Process
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# First off, taking a *step back* and drawing out the xy plane and the directions the are required was helpful. 
+# 
+#  .. image:: Power_of_thor_pic_2.png
+#  
+# However, without the original code telling me exactly what the directions did, I had no idea of knowing that the initial Thor position never changed. Keeping notes helped me track how past codes I wrote worked and didn’t work. Taking notes was helpful then just trying to type in code without much guidance. It will take me a while to look and gather before trying code. 
+# I started with trying to figure out Thor's position in comparison to light's position but did it have in common with directions.
```
Comment (bjones1) [inline Thor Method.py:75]: ??? What does "but did it have in common with directions." mean?

File: Thor Method.py [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+#
+# .. image:: Power_of_thor_pic_4.png
+#
+# I started with a while loop but noticed it only worked for a straight line movement. Next i tried if loops. That just got messy. Last i tried a for loop, and incorprated Thor's position getting updated.
```
Comment (bjones1) [inline Thor Method.py:79]: Avoid history -- tell us the end result of what you learned.

File: Thor Method.py [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+# ^^^^^^^^^^^^^^^^^
+# Looking back, if I was going to do the game in C++, I really should have done the codechat in C++ also. I am learning Python now to make this a little easier on me so I don't have to switch back and forth. I would copy and paste the C++ code I used in the game; however, python doesn't care for C++ and keeps putting errors at the top. 
+#
+# Soultion: Second Try with Python
+# ----------------------------------------
+# With this second try, I am going to work in `Python <https://wiki.python.org/moin/BeginnersGuide>`_.
+#
+# My Thoughts and Process
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# The whole idea is to get Thor to a power source using the xy plane. So if Thor is at (0,3) and the light is at (1,3) then Thor needs to move to the right one. Since postive x is to the right (**East**) and postive y is down (**South**). Another given is that x is 0-40 and y is 0-18 and the coodinates start at the top left of the boundry.
```
Comment (bjones1) [inline Thor Method.py:98]: Good! Refer to a figure to help us see how coordinates correspond to compass directions.

File: Thor Method.py [hunk 10]
```diff
@@ -0,0 +1,224 @@
...
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# The whole idea is to get Thor to a power source using the xy plane. So if Thor is at (0,3) and the light is at (1,3) then Thor needs to move to the right one. Since postive x is to the right (**East**) and postive y is down (**South**). Another given is that x is 0-40 and y is 0-18 and the coodinates start at the top left of the boundry. 
+#
+# However, as I found out this my first try, Thor's position isn't being tracked, just his intial position. This needs to taken into account. We already found out with try 1 that loops aren't the best way to complete this code. Maybe we can use math instead. 
+#
+#  Example: 
+#
+#   If Thor is at (0,0) and the light is at (40,0). We know just by looking at this that Thor is -40 from the light in the x direction. So to solve this we need to add 40 to Thor's intial position.
+#
+print(initial_tx, initial_ty, light_x, light_y)
```
Comment (bjones1) [inline Thor Method.py:106]: Is this for debug output? If not, it's not in the right form...

File: Thor Method.py [hunk 11]
```diff
@@ -0,0 +1,224 @@
...
+#
+#  Example: 
+#
+#   If Thor is at (0,0) and the light is at (40,0). We know just by looking at this that Thor is -40 from the light in the x direction. So to solve this we need to add 40 to Thor's intial position.
+#
+print(initial_tx, initial_ty, light_x, light_y)  
+#
+#| The above code gets the program to show us where Thor and the light start.
+#
+x_steps = [int]  # this is going to track our difference in Thor's x and Light's x.
```
Comment (bjones1) [inline Thor Method.py:110]: These two lines make no sense to me. Also, `x_steps` and `y_steps` are reassigned in line 111-112, so these two statements are also unnecessary.

File: Thor Method.py [hunk 12]
```diff
@@ -0,0 +1,224 @@
...
+y_steps = [int]  # this is going to track our difference in Thor's y and Light's y.
+
+x_steps = initial_tx - light_x  # examples:  -40 = 0 - 40 (E); 40 = 40 - 0 (W)
+y_steps = initial_ty - light_y  # examples:  -18 = 0 - 18 (S) ; 18 = 18 -0 (N)
+
+if x_steps == 0:
```
Comment (bjones1) [inline Thor Method.py:116]: Explain this! It's the core of your algorithm!

File: Thor Method.py [hunk 13]
```diff
@@ -0,0 +1,224 @@
...
+    if x_steps < 0:
+        print("E")
+        x_steps += 1
+
+# So I spent almost an hours tryin to get that version to work however it kept giving me a time out error. So instead of working with new varibles. Lets try working with the ones we already have.
+#
+# Let's try increment Thor's initial position with every move. 
+#
+# game loop
+while True:
```
Comment (bjones1) [inline Thor Method.py:136]: I'm confused again -- it seems like you have 3 or 4 programs in this program? Just present your final results.

File: Thor Method.py [hunk 14]
```diff
@@ -0,0 +1,224 @@
...
+    remaining_turns = int(input())  # The remaining amount of turns Thor can move. Do not remove this line.
+
+    # Write an action using print
+    # To debug: print("Debug messages...", file=sys.stderr, flush=True)
+
+    #
+    #| The above code gets the program to show us where Thor and the light start.
```
Comment (bjones1) [inline Thor Method.py:143]: The use of `#| Comments here` produces a monospaced line -- why do this?

**Review by bjones1 — CHANGES_REQUESTED** [review]

**Comment by bjones1** [comment]
I'm assuming this is outdated? If so, please close this PR. It not, close all the other PRs.

File: Thor Method.py (hunk 3 above)
Comment (Starnol) [reply Thor Method.py:50]: That was the given code. And since I am just learning python... I have no idea.

File: Thor Method.py (hunk 4 above)
Comment (Starnol) [reply Thor Method.py:52]: its part of the given code..

File: Thor Method.py (hunk 5 above)
Comment (Starnol) [reply Thor Method.py:61]: That was the given code. think I removed it in the end.

File: Thor Method.py (hunk 10 above)
Comment (Starnol) [reply Thor Method.py:106]: it was to debug but it didn't work so I removed it from my final code.

File: Thor Method.py (hunk 11 above)
Comment (Starnol) [reply Thor Method.py:110]: I was keeping track of my thought process. this one didn't work

File: Thor Method.py (hunk 12 above)
Comment (Starnol) [reply Thor Method.py:116]: I was trying to get how far thor was from the light. IT didn't work.

File: Thor Method.py (hunk 13 above)
Comment (Starnol) [reply Thor Method.py:136]: I went about it different when I realized then steps didn't work.

File: Thor Method.py (hunk 14 above)
Comment (Starnol) [reply Thor Method.py:143]: Think that was part of the given code.

**Comment by Starnol** [comment]
Will try to clean it up some but alot of your comments was about the given code.

**Pull request closed without merging**